action.yml

```diff
@@ -79,6 +79,75 @@ runs:
         aws_secret_access_key: ${{ inputs.aws_secret_access_key }}
         aws_region: ${{ inputs.aws_region }}
 
+    - name: Render and deploy task definition to Amazon ECS
```

Comment:
Is it possible to create a new action for the preflight stuff?
Like `moneymeets/action-ecs-deploy/custom-deploy-steps/run-preflight-cointainer` or something else and put everything related to this action?

Background:
It could be the case that we create an app without the preflight container in the future (softfair-proxy, keycloak, mail-dispatcher).
So we could introduce a new input like `preflight: true`, or `preflight: false` to our ecs-deploy action and set it where the action is called. But I don't want to think about every single step if the condition for preflight is needed or not. If we move the preflight stuff to an isolated action, we just need the condition ones.

Reply:
This seems like a good idea

Reply:
https://moneymeets.atlassian.net/browse/MD-7491